fdbserver/MoveKeys.actor.cpp

```diff
@@ -180,31 +186,54 @@ ACTOR Future<Void> deleteCheckpoints(Transaction* tr, std::set<UID> checkpointId
 }
 } // namespace
 
-bool DDEnabledState::isDDEnabled() const {
-	return ddEnabled;
+bool DDEnabledState::sameId(const UID& id) const {
```

Comment:
[Optional] This function is presumably added for use by other classes. But, there are many locations in the implementation below that compare the `UID`s -- it might make sense to also use this function there for slightly simpler code.